crux/src/Crux/Config/Solver.hs

```diff
@@ -65,6 +66,9 @@ data SolverConfig = SingleOnlineSolver SolverOnline
                   | OnlyOfflineSolver SolverOffline
                   -- ^ Use only an offline solver with no support for path
                   -- satisfiability checking
+                  | OnlyOfflineSolvers [SolverOffline]
```

Comment:
Does `OnlyOfflineSolver` need to be a separate case?

Reply:
Not necessarily, I'm ambivalent about whether or not this should be a separate case or to just use pattern matching. I can remove the constructor.